src/api/config/routes.rb

```diff
@@ -188,10 +188,12 @@ def self.public_or_about_path?(request)
       post 'project/create_image_repository' => :create_image_repository
 
       # Flags
+      post 'flag/manipulate/:project(/:package)' => :manipulate_flag, constraints: cons, as: 'manipulate_repository_flag'
+      # TODO bento_only
       put 'flag/:project(/:package)' => :toggle_flag, constraints: cons, as: 'toggle_repository_flag'
       post 'flag/:project(/:package)' => :create_flag, constraints: cons, as: 'create_repository_flag'
       delete 'flag/:project(/:package)/:flag' => :remove_flag, constraints: cons, as: 'remove_repository_flag'
-      post 'flag/:project(/:package)' => :manipulate_flag, constraints: cons, as: 'manipulate_repository_flag'
+      # end TODO
```

Comment:
I am not sure if this is clear, I would add the routes names to the first comment instead :thinking:

Reply:
I just repeat the todo